source/win-main/DocumentTabs.vue

```diff
@@ -514,10 +534,21 @@ body.linux {
 
       div[role="tab"] {
         border-color: rgb(120, 120, 120);
-        background-color: #5a5a5a;
+        background-color: rgb(40, 40, 40);
+        cursor: pointer;
+
+        &:hover { background-color: rgb(45, 45, 45); }
+        &.active {
+          background-color: rgb(50, 50, 50);
+        &.hover {
+          background-color: rgb(55, 55, 55);
+        }
+          }
+
+      &:not(:last-child) {
+        border-right: 1px solid rgb(30, 30, 30);
+      }
```

Comment:
As I mentioned in my comment in the Discussions section, we might get away with merging the Linux section into the generic body section and then simply resetting the unwanted styles for Windows and macOS.